src/AsyncGenerator.yml

```diff
@@ -0,0 +1,249 @@
+﻿solution:
+  filePath: ..\..\..\src\NHibernate.sln #wants to resolve from the execuing path.
```

Comment:
Thanks,
And I only realize now that those typo were in NHibernate yaml too. But well, better have a good published code wording. That is less an issue for custom code in a configuration file.